src/ribasim_nl/tests/test_cloud.py

```diff
@@ -0,0 +1,59 @@
+import shutil
+from pathlib import Path
+
+import pytest
+from ribasim_nl import Cloud
+
+DATA_DIR = Path(__file__).parent.joinpath("data", "cloud")
+if DATA_DIR.exists():
+    shutil.rmtree(DATA_DIR)
+
```

Comment:
Shall we use the `tmp_path` fixture here? That could be fed as an argument to the `cloud` fixture: `def cloud(tmp_path)`.

https://docs.pytest.org/en/7.1.x/reference/reference.html?highlight=tmp_path#std-fixture-tmp_path

Gets cleaned up automatically.